### DISCUSSION TITLE
weblate hosting with other sites

### DISCUSSION THREAD

**Discussion by harishsubash** [WeblateOrg/weblate#5509] [Q&A] [answered]
I hosted weblate using nginx + uwsgi virtualenv + ubuntu as in the documentation. 

This Server will host multiple sites like example.com/weblate and example.com/app2
Here are my nginx and uwsgi configs

`

    server {
    listen 80 default_server;
    listen [::]:80 default_server;
    server_name example.com;

    location ~ ^/favicon.ico$ {
        # DATA_DIR/static/favicon.ico
        alias /home/ubuntuuser/weblate-env/lib/python3.6/site-packages/weblate/static/favicon.ico;
        expires 30d;
    }

    location /weblate/static/ {
        # DATA_DIR/static/
        alias /home/ubuntuuser/weblate-env/lib/python3.6/site-packages/weblate/static/;
        expires 30d;
    }

    location /weblate/media/ {
        # DATA_DIR/media/
        alias /home/ubuntuuser/data/media/;
        expires 30d;
    }

    location /weblate/ {
	include uwsgi_params;
        # Needed for long running operations in admin interface
        uwsgi_read_timeout 3600;
        # Adjust based to uwsgi configuration:
        #uwsgi_pass unix:///run/uwsgi/app/weblate/socket;
        uwsgi_pass 127.0.0.1:8080;
    }
}
`

`[uwsgi]
plugins       = python3
master        = true
protocol      = uwsgi
socket        = 127.0.0.1:8080
wsgi-file     = /home/azureuser/weblate-env/lib/python3.6/site-packages/weblate/wsgi.py

virtualenv = /home/ubuntuuser/weblate-env

buffer-size   = 8192

reload-on-rss = 250

workers       = 8

enable-threads = true

close-on-exec = true

umask = 0022

uid = ubuntuuser
gid = ubuntuuser

ignore-sigpipe = true
ignore-write-errors = true
disable-write-exception = true
`
When I go to the site using example.com/weblate it is showing page not found from weblate. How can I redirect /weblate to the root site of the weblate.

**Comment by nijel** [comment, marked as answer]
You're probably missing:

```
URL_PREFIX = "/weblate"
```

See https://docs.weblate.org/en/latest/admin/install.html#running-weblate-under-path

**Reply by harishsubash** [reply]
Thanks nijel, it worked.
